conda-store-server/environment-dev.yaml

```diff
@@ -7,57 +7,22 @@ dependencies:
   - python ==3.10
   # conda builds
   - conda ==23.5.2
-  - conda-docker >= 0.1.2
   - python-docker
-  - conda-pack
-  - conda-lock >=1.0.5
-  - conda-package-handling
-  - conda-package-streaming
   # web server
-  - celery
   - flower
-  - redis-py
-  - sqlalchemy<=1.4.47
-  - psycopg2
-  - pymysql
-  - requests
-  - uvicorn
-  - fastapi
-  - pydantic < 2.0
```

Comment:
Same here. The dependencies need to be reviewed carefully. For instance, this file is used by integration tests to run the test code. Currently, the integration tests don't run at all - they fail during the collect step because a different version of pydantic is being used (instead of the <2 version specified above).

From the test output:

```
==================================== ERRORS ====================================
______________________ ERROR collecting tests/test_api.py ______________________
../tests/test_api.py:25: in <module>
    from conda_store_server import schema
conda_store_server/schema.py:50: in <module>
    class AuthenticationToken(BaseModel):
conda_store_server/schema.py:55: in AuthenticationToken
    role_bindings: Dict[constr(regex=ARN_ALLOWED), List[str]] = {}
E   TypeError: constr() got an unexpected keyword argument 'regex'
=========================== short test summary info ============================
ERROR ../tests/test_api.py - TypeError: constr() got an unexpected keyword argument 'regex'
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.34s ===============================
```

The workflow code:

```yaml
  integration-test-conda-store-server:
...
        with:
          environment-file: conda-store-server/environment-dev.yaml
```

I suspect there will be other failures once you add pydantic back because the tests import `conda_store_server`.